app.js

```diff
@@ -26,73 +26,75 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(methodOverride("_method"));
-app.use(fileUpload({
-  limits: { fileSize: 1024 * 1024 }
-}));
+app.use(
+  fileUpload({
+    limits: { fileSize: 1024 * 1024 },
+  })
+);
 
 app.use(
-    session({
-        key: "user_sid",
-        secret: "super_secret",
-        resave: false,
-        saveUninitialized: false,
-        cookie: {
-            expires: 600000,
-        },
-    })
+  session({
+    key: "user_sid",
```

Comment:
is this a typo for user_id?

Reply:
it was like that in the template :) 'sid' may be for "secret identifier"

Reply:
This is the correct answer I believe